src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      >
+      <label>
+        <div class="bigBox">
+          <img
+            class="bigIcon"
+            src="./images/Search.svg"
+            alt="search"
+            >
+          <input
+            class="searchBar big"
+            id="bigSearch"
+            data-qa="keypress"
+            type="search"
+            name="searchBar"
+            placeholder="Try &quot;Los Angeles&quot;"
+            >
+        </div>
+      </label>
+    </form>
+    <form
```

Comment:
```suggestion
    </form>
    
    <form
```
put space between blocks of code